docs/topics/downloader-middleware.rst

```diff
@@ -1073,6 +1073,8 @@ In order to use this parser:
 * Set :setting:`ROBOTSTXT_PARSER` setting to
   ``scrapy.robotstxt.ReppyRobotParser``
 
+* Only works with python 3.8 and earlier
```

Comment:
I suggest we put this at the beginning of the section, as a warning:

```rst
.. warning:: Does not support Python 3.9+
```

Reply:
@Gallaecio  did you mean just before the install?

Reply:
If we keep both of them then it will be more informative I guess.